sys-process/oomd/oomd-0.5.0-r1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+SRC_URI="https://github.com/facebookincubator/oomd/releases/tag/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/facebookincubator/oomd/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64"
+
+inherit meson
+
+DESCRIPTION="oomd is userspace Out-Of-Memory (OOM) killer for linux systems."
+HOMEPAGE="https://github.com/facebookincubator/oomd"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="man systemd"
+DEPEND="dev-libs/jsoncpp:=
+	systemd? ( sys-apps/systemd )"
+RDEPEND="${DEPEND}"
+CONFIG_CHECK="~CONFIG_PSI"
+ERROR_PSI="CONFIG_PSI is required for oomd to function"
+
+src_configure() {
+	local emesonargs
+	emesonargs=(
+		$(meson_feature man)
+		$(meson_feature systemd)
```

Comment:
What does USE=systemd do here?

Reply:
I'm not sure if I got the USE flag for systemd correct, `oomd` can optionally be build with systemd support via libsystemd.